### ISSUE TITLE
Nothing to publish for tag []

### ISSUE THREAD

**Issue by unidotnet** [Torann/laravel-geoip#37]
Hey ,
When I tried the following command. There is alway no config file generated.

Laravel version 5.1 :

php artisan vendor:publish
Nothing to publish for tag [].

php artisan vendor:publish --provider=Torann\GeoIP\GeoIPServiceProvider --tag=config
Nothing to publish for tag [config].

php artisan vendor:publish --provider="Torann\GeoIP\GeoIPServiceProvider" --force
Nothing to publish for tag [].

**Comment by bastiendonjon** [comment]
+1

**Comment by cringer** [comment]
unidotnet,

`php artisan vendor:publish`, `php artisan vendor:publish --force` and `php artisan vendor:publish --provider="Torann\GeoIP\GeoIPServiceProvider" --force` should work.

`php artisan vendor:publish --provider=Torann\GeoIP\GeoIPServiceProvider --tag=config` will not work as there is no file group defined in the service provider.

If it is still not working for you please make sure that it is installed correctly for Laravel 5.1 as the README is 5.0

```
'providers' => [
    ...
    Torann\GeoIP\GeoIPServiceProvider::class,
]
```

```
'aliases' => [
    ...
    'GeoIP'     => Torann\GeoIP\GeoIPFacade::class,
]
```

**Comment by Rjs37** [comment]
It would be good if this could be fixed so that we can publish just this packages config. Rather than being forced to publish everything. 

Especially when the merge command isn't used here either, meaning that I need to publish this config for it to work.

**Comment by Torann** [comment]
This is fixed in v1.0
